lib/rss_feed.rb

```diff
@@ -0,0 +1,30 @@
+class RssFeed
+  require 'rss'
```

Comment:
**note** mostly a note to myself - rss is not included in the standard library for Ruby 3.0, and needs to be added to the Gemfile. I had fixed this in the previous location but will need to update. ref https://github.com/meedan/check-api/pull/1511